src/css/style.css

```
@@ -7,6 +7,32 @@
   --button-light-blue: #76A4FF;
   --input-border-blue: #3878F9;
   --gray-op-2: rgb(139, 148, 158, 0.2);
+  --radius: 10px;
+}
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+.d-none {
+  display: none !important;
+}
+.mt-2 {
+  margin-top: 0.75rem !important;
+}
+.mt-3 {
+  margin-top: 1rem !important;
+}
+.mt-6 {
+  margin-top: -4rem !important;
+}
```

what do you think about use:

```suggestion
.-mt-16 {
  margin-top: -4rem !important;
}
```
To make reference that is a negative valor

Classes | Values
-- | --
-mt-16  | margin-top: -4rem; /* -64px */

---

If doesn't work you can try:

```suggestion
.\-mt-16 {
  margin-top: -4rem !important;
}
```